src/main/kotlin/skytils/skytilsmod/features/impl/misc/ItemFeatures.kt

```diff
@@ -478,6 +478,22 @@ class ItemFeatures {
         getExtraAttributes(item)?.let { extraAttributes ->
             if (Skytils.config.showPotionTier && extraAttributes.hasKey("potion_level")) {
                 stackTip = extraAttributes.getInteger("potion_level").toString()
+            } else if (Skytils.config.showShardLevel && item.item == Items.prismarine_shard && extraAttributes.hasKey("attributes")) {
+                /*If they ever add the ability to combine attributes on shards, this will need to be updated to:
```

Comment:
Shouldn't it be like enchanted books where it only displays the level if only one attribute is present? Otherwise it could get confusing in my opinion